src/BackgroundPublish.h

```diff
@@ -26,9 +26,14 @@
 #include "Particle.h"
 
 using publish_completed_cb_t = std::function<void(particle::Error status,
+    const char *event_name,
```

Comment:
nit, `char*` instead of `char *`

Reply:
This seems all over the place in our code, so I'd wait until there's a style document and we can just run everything through `clang-format`